cmake/ElektraCache.cmake

```diff
@@ -37,7 +37,7 @@ set (ADDED_DIRECTORIES "" CACHE STRING ${PLUGINS_DOC} FORCE)
 
 remember_for_removal(BINDINGS TO_REMOVE_BINDINGS)
 
-set (BINDINGS_LIST_DEFAULT cpp)
+set (BINDINGS_LIST_DEFAULT cpp;glib;gsettings)
```

Comment:
thats better in a different PR. Maybe we should even change to ALL, when dependencies are missing they are kicked out anyway.